src/test/java/com/atlassian/shele/shelePlugin/ao/test/RestIssueTest.java

```diff
@@ -0,0 +1,40 @@
+package com.atlassian.shele.shelePlugin.ao.test;
+
+import com.atlassian.shele.shelePlugin.ao.ProjectDTO;
+import com.atlassian.shele.shelePlugin.ao.ProjectEntityService;
+import com.google.common.collect.ImmutableList;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.List;
+
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.BDDMockito.given;
+
+/**
+ * Unit test on RestIssueService
+ */
+public class RestIssueTest {
+
+    @Mock
+    private ProjectEntityService service;
+
+    @Before
+    public void setUp() throws Exception {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    //Test have passed but appeared some warnings from mockito
```

Comment:
You can reword comment like `//TODO: fix warnings` if you don't have ideas how to fix it